core/shared/src/main/scala/scalaz/zio/ZSchedule.scala

```diff
@@ -721,18 +758,22 @@ object ZSchedule extends Schedule_Functions {
 
   implicit val ConformsAnyProof: ConformsR[Any] = _ConformsR1
 
-  sealed case class Decision[+A, +B](cont: Boolean, delay: Duration, state: A, finish: () => B) { self =>
+  sealed case class Decision[+A, +B](cont: Boolean, delay: Delay, state: A, finish: () => B) {
```

Comment:
This should probably be `final case class` rather than `sealed case class` (I know you didn't change this, I just saw it 😄 ).

Reply:
No problem, I'll change it :)